### DISCUSSION TITLE
2.4.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#354] [Releases]
- Improve card list for landscape and tablet display
- Add theming colour support (thanks, Subhashish Anand!)
- Don't close scan activity on camera error (so manual entry is still possible)
- Add all contributors to the about dialog

<hr /><em>This discussion was created from the release <a href='https://github.com/TheLastProject/Catima/releases/tag/v2.4.0'>2.4.0</a>.</em>

**Comment by pxn** [comment]
What happened to F-droid releases?

**Reply by TheLastProject** [reply]
I care greatly about F-Droid and I'm even part of the F-Droid team. I still ensure that Catima will update on F-Droid.

The issues are sadly on F-Droid's side, there is only one person who can sign the APKs that get published on F-Droid (F-Droid build from source themselves) and that person has just been having less time lately. Because of that, it can sometimes take several days longer than expected for an update to show up on F-Droid.

Rest assured, I have no plans whatsoever to stop supporting F-Droid and this release will become available on F-Droid as well.

**Reply by pxn** [reply]
Great. Thanks for update.
